assets/templates/command/install_server.md

```diff
@@ -0,0 +1,20 @@
+{{ .ConfluenceURL }} has been successfully added. To finish the configuration, add an Application Link in your Confluence instance following these steps:
+
+1. Navigate to [**Settings > Applications > Application
+   Links**]({{ .ConfluenceURL }}/plugins/servlet/applinks/listApplicationLinks)
+2. Click **Create link**.
+3. In **Create Link** screen, select **External Application** and **Incoming** as
+   `Application type` and `Direction` respectively. Click **Continue**.
+4. In **Link Applications** screen, set the following values:
+**Name**: `Mattermost`
+**Redirect URL**: ```{{ .RedirectURL }}```.
+**Application Permissions**: `Admin`
+Click **Continue**
+5. Copy the `clientID` and `clientSecret` and add them into the plugin configuration.
+6. Use the "/confluence connect" command to connect your Mattermost account with your
+   Confluence account.
+7. Use the "/confluence connect URL admin" for confluence admin only to save the token for the first time only.   
+8. Click the "More Actions" (...) option of any message in the channel
+   (available when you hover over a message).
```

Comment:
this line is no longer needed

Reply:
Just to confirm you are taking about point number 8 and line 17,18 which starts with Click the "More Actions " ...

Reply:
Yes, thanks!